config/base.json

```
@@ -35,6 +35,7 @@
 	},
 	"flags": {
 		"enable_datadog": true,
-		"enable_io_beta_cta": true
+		"enable_io_beta_cta": true,
+		"enable_hvd_on_hvd_preview_branch": false
```

This is a total nit, but what if it was this:
```suggestion
		"enable_hvd_on_preview_branch": false
```
slightly less verbose, but not a big deal either way IMO

---

K fixing in a new commit.